components/service/sync-logins/src/main/java/mozilla/components/service/sync/logins/AsyncLoginsStorage.kt

```diff
@@ -111,9 +111,21 @@ interface AsyncLoginsStorage : AutoCloseable {
      *
      * @throws InvalidKeyException if the encryption key is wrong, or the db is corrupt
      * @throws MismatchedLockException if we're already unlocked
+     * @throws LoginsStorageException On unexpected errors (IO failure, rust panics, etc)
      */
```

Comment:
Perhaps we actually want to use `@Throws` annotation?
> This annotation indicates what exceptions should be declared by a function when compiled to a JVM method.

Reply:
I'll rename it to @rejectsWith, or something.